manifests/m/monero/monero-gui/1.1/monero.monero-gui.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: monero.monero-gui
+PackageVersion: 1.1

```

```suggestion
PackageVersion: 0.18.1.1
```
This must match Control Panel.